sentry_sdk/tracing.py

```diff
@@ -1156,6 +1161,170 @@ def _set_initial_sampling_decision(self, sampling_context):
         pass
 
 
+class POTelSpan:
+    """
+    OTel span wrapper providing compatibility with the old span interface.
+    """
+
+    # XXX Maybe it makes sense to repurpose the existing Span class for this.
+    # For now I'm keeping this class separate to have a clean slate.
+
+    # XXX The wrapper itself should have as little state as possible
+
+    def __init__(
+        self,
+        active=True,  # type: bool
+        trace_id=None,  # type: Optional[str]
+        span_id=None,  # type: Optional[str]
+        parent_span_id=None,  # type: Optional[str]
+        same_process_as_parent=True,  # type: bool
+        sampled=None,  # type: Optional[bool]
+        op=None,  # type: Optional[str]
+        description=None,  # type: Optional[str]
+        status=None,  # type: Optional[str]
+        containing_transaction=None,  # type: Optional[Transaction]
+        start_timestamp=None,  # type: Optional[Union[datetime, float]]
+        scope=None,  # type: Optional[sentry_sdk.Scope]
+        origin="manual",  # type: str
+    ):
+        # type: (...) -> None
+        from sentry_sdk.integrations.opentelemetry.consts import SentrySpanAttribute
+
+        self._otel_span = tracer.start_span(description or op or "")  # XXX
+        self._active = active
+
+        self._otel_span.set_attribute(SentrySpanAttribute.ORIGIN, origin)
+        if op is not None:
+            self._otel_span.set_attribute(SentrySpanAttribute.OP, op)
+        if description is not None:
+            self._otel_span.set_attribute(SentrySpanAttribute.DESCRIPTION, description)
+
+    def __enter__(self):
+        # type: () -> POTelSpan
+        # XXX use_span? https://github.com/open-telemetry/opentelemetry-python/blob/3836da8543ce9751051e38a110c0468724042e62/opentelemetry-api/src/opentelemetry/trace/__init__.py#L547
```

Comment:
i'd prefer `use_span` here too because it does a bit of exception/status logic on its own